UnitTests/Input/ResponderTests.cs

```diff
@@ -129,4 +130,124 @@ public void IsOverridden_True_IfOverridden ()
 		Assert.Empty (Responder.Instances);
 #endif
 	}
+
+	[Fact]
+	public void Responder_Not_Notifying_Dispose ()
+	{
+		List<View> views = new List<View> ();
+		var container1 = new View () { Id = "Container1" };
+
+		for (int i = 0; i < 20; i++) {
```

Comment:
Is there really any value in doing these tests 20 times vs just once? 

This seems like un-needed complexity in the unit tests.

If there is a specific edge case that needs to be tested, then a specific unit test that tests that should be provided.

Reply:
I'll change it to only test a single view for all.